src/TaskSet.php

```diff
@@ -72,6 +72,8 @@ public function set(array $args): void
 		foreach ($input->getOptions() as $key => $value) {
 			// Black magic. It parse --database.host=localhost into $tmp array.
 			// Easy to use! :-)
+			/** @var string $value */
+			$value = $value;
```

Comment:
Stacilo by `parse_str('parameters[' . str_replace('.', '][', $key) . ']=' . (string) $value, $tmp);`

Reply:
OK. Zkusím.

Reply:
```
 ------ --------------------------------------------------------- 
  Line   TaskSet.php                                              
 ------ --------------------------------------------------------- 
  75     Cannot cast array|bool|float|int|string|null to string.  
 ------ --------------------------------------------------------- 
```

Reply:
https://github.com/petrparolek/contributte-neonizer/actions/runs/1662846839

Reply:
Tak to chce vymyslet neco jineho. `$value = $value;` je blbost.

Reply:
Nejlép nám poradí @ondrejmirtes

Reply:
Ahoj, prosim, jestli chcete poradit, zreprodukujte problem na phpstan.org, a idealne se zeptejte anglicky v Discussions :) https://github.com/phpstan/phpstan/discussions Diky.

Reply:
Nezlob se @ondrejmirtes. :-)

Reply:
Ja se nezlobim, ale musim nejak managovat svuj cas, navic na anglickou diskuzi muzu pozdeji linkovat kdyz se nekdo zepta na neco podobneho 😊

Reply:
To ja uplne chapu. Uz jsem domluvil Petrovi, aby te neoznacoval u kravin. :-)